py/modthread.c

```diff
@@ -154,6 +154,8 @@ typedef struct _thread_entry_args_t {
     mp_obj_t args[];
 } thread_entry_args_t;
 
+extern const mp_print_t mp_stderr_print;
```

Comment:
I'm guessing that you will not want the `extern` here.

I was not sure where to put it since `mp_stderr_print` is defined in `main.c`.